frontend/src/scenes/insights/filters/BreakdownFilter/index.ts

```diff
@@ -1 +1,5 @@
 export { BreakdownFilter } from './TaxonomicBreakdownFilter'
+
+export const isURLNormalizeable = (propertyName: string): boolean => {
+    return ['$current_url', '$pathname'].includes(propertyName)
```

Comment:
perhaps also `propertyName.endsWith('_url')`?

Reply:
I considered that but I think the next step would be to add a property type of URL and _maybe_ autodetect that